File: Aula05/exemplo.cpp
#include <iostream>
using namespace std;

void exibirMenu(){
    cout << "=== MENU ===\n";
    cout << "1 - Cadastrar produto\n";
    cout << "2 - Listar produtos\n";
    cout << "3 - Sair\n";
}

int main(){
    int opcao;

    while(true){
        exibirMenu();
        cin >> opcao;

        if(opcao == 3){
            break;
        }
    }
}

File: Aula05/exemplo.java
import java.util.Scanner;

public class Menu {
    public static void exibirMenu(){
        System.out.println("=== MENU ===");
        System.out.println("1 - Cadastrar produto");
        System.out.println("2 - Listar produtos");
        System.out.println("3 - Sair");
    }

    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int opcao;

        while(true){
            exibirMenu();
            opcao = sc.nextInt();

            if(opcao == 3){
                break;
            }
        }
    }
}